src/symbol/shaping.js

```diff
@@ -35,17 +35,20 @@ export type PositionedGlyph = {
 
 // A collection of positioned glyphs and some metadata
 export type Shaping = {
-    positionedGlyphs: Array<PositionedGlyph>,
+    positionedGlyphs: Array<Array<PositionedGlyph>>,
```

Comment:
I got a flow error that I don't understand, @ansis @asheemmamoowala , any hints about it?
```
Error -----src/symbol/shaping.js:38:35



`PositionedGlyph` [1] is incompatible with array type [2].

   src/symbol/shaping.js:38:35
   38|     positionedGlyphs: Array<Array<PositionedGlyph>>,
                                         ^^^^^^^^^^^^^^^ [1]

References:
   src/symbol/shaping.js:38:29
   38|     positionedGlyphs: Array<Array<PositionedGlyph>>,
                                   ^^^^^^^^^^^^^^^^^^^^^^ [2]
```

Reply:
I'm not sure exactly what is going on but I think it isn't compatible with the typing of `isEmpty`. Adding `any` to `function isEmpty(array: any) {` seems to avoid this. But maybe adding real typing to `isEmpty` would be better

Reply:
Thank you very much!